### PR TITLE
Use redux toolkit in exercises

### LINKED ISSUE #26: Rewrite redux-sauce to Redux Toolkit
Since we [agreed](https://www.notion.so/salsita/2020-07-03-Training-Crew-Sync-afd46e512bd54dcbb3ab8d7f6537fedc) that [redux toolkit](https://redux-toolkit.js.org/) is our official way to write redux actions, we need to update the training materials.

This will include:

1. Rewrite of the solution for the redux sauce exercise
2. Update the solution in _all_ consecutive exercises
3. Update [Tutorial](https://github.com/salsita/react-training/blob/master/README.md)
4. Update Slides in the presentation

Please check the linked pull request with the skeleton of the change (only step 1 was done)

### PR DESCRIPTION
**Pull request by dinassh**

### CODE REVIEW

File: exercises/06-redux-saga/src/modules/users/components/users-list.js [hunk 1]
```diff
@@ -51,8 +51,8 @@ const mapStateToProps = state => ({
   users: UsersSelectors.getUsersList(state)
 });
 
-const mapDispatchToProps = dispatch => ({
-  addUser: user => dispatch(UsersActions.addUser(user))
-});
+const mapDispatchToProps = {
```
Comment (novellizator) [inline exercises/06-redux-saga/src/modules/users/components/users-list.js:54]: this is naturally how people use it in client projects. The question is whether we don't want to keep it in an expanded form for educational purposes. Or have it expanded i nthe first exercise and then start using this? thoughts @dinassh @branoholy ?

File: README.md [hunk 2]
```diff
@@ -250,7 +250,7 @@ Props:
 ### Users slice
 Location: `src/modules/users/users-slice.js`
 
-The same actions and reducer, but created with `Redux Toolkit`.
+The same action and reducer, but created with `Redux Toolkit`.
```
Comment (novellizator) [inline README.md:253]: why singular?

File: exercises/06-redux-saga/src/modules/users/components/users-list.js (hunk 1 above)
Comment (dinassh) [reply exercises/06-redux-saga/src/modules/users/components/users-list.js:54]: Both options are described in the slides, see pages 33 and 34. The expanded version is not currently used in solutions. 

EDIT: it's on pages 37 and 38 now

File: README.md (hunk 2 above)
Comment (dinassh) [reply README.md:253]: Because only one action called `ADD_USER` exists at this point

**Pull request closed without merging**